.tekton/install-node.sh

```diff
@@ -0,0 +1,27 @@
+#!/bin/bash
+
+PRERELEASE_NODE_VERSION="$1"
+
+echo "Installing node prerelease version $PRERELEASE_NODE_VERSION.."
```

Comment:
issue: I cannot find this log in your [CI build.](https://cloud.ibm.com/devops/pipelines/tekton/c2cd6a8d-ea5a-47b0-913e-cd172d63833f/runs/0e90b7f8-6452-4027-892f-2d92c27251fc/test-ci-metrics-util-task/run-test-group?env_id=ibm:yp:eu-de)

Reply:
That is the pr listener you are checking. In the PR listener `prerelease` attribute if not set.

You can find the logs in here [node-prerelease-pipeline CI](https://cloud.ibm.com/devops/pipelines/tekton/c2cd6a8d-ea5a-47b0-913e-cd172d63833f/runs/97cffca3-2f00-4f1c-bc20-b283c27eee3f/test-ci-aws-lambda-task/run-test-group?env_id=ibm:yp:eu-de)


I will run the pipeline again after the pr changes, these seems to be multiple log statments present. I will update that.

Reply:
FYI: when implemented `install-node.sh`, there seems to be an issue with 20 is also being installed in the pipeline. That is unnecessary. I will resolve that as well. Putting it here for visibility